azurelinuxagent/ga/exthandlers.py

```diff
@@ -304,10 +319,52 @@ def cleanup_outdated_handlers(self):
             if os.path.isfile(pkg):
                 try:
                     os.remove(pkg)
-                    logger.verbose("Removed extension package {0}".format(pkg))
-                except OSError as e:
-                    logger.warn("Failed to remove extension package {0}: {1}".format(pkg, e.strerror))
-   
+                    logger.verbose("Removed extension package "
+                                "{0}".format(pkg))
+                except Exception as e:
+                    logger.warn("Failed to remove extension package: "
+                                "{0}".format(pkg))
+    
+    #TODO: Why is this here with extension handlers... this doesn't seem right.
+    def handle_remote_access(self):
+        logger.verbose("Entered handle_remote_access")
+        self.protocol.client.update_goal_state(True)
+        self.protocol.client.update_remote_access_conf(self.protocol.client.goal_state)
+        remote_access = self.protocol.client.remote_access
+        if remote_access is not None:
+            # Get existing users.
+            osUtils = get_osutil()
+            existingUsers = osUtils.getusers()
+            userNames = []
+            for usr in existingUsers:
```

Comment:
How is this list filtered?  How do you know everyone is eligible?

Reply:
This has changed now.